README.md

```diff
@@ -1,160 +1,9 @@
-
-  ============================================================
-```
-
-#### Viewing your results
-
-Once an analysis has been uploaded, you can view your results in the FOSSA web application. You can see your analysis by using the link provided as output by `fossa analyze`, or by navigating to your project and revision in the FOSSA web application.
-
-#### What next?
-
-Now that your analysis is complete, there are a couple things you might want to do after an initial integration:
-
-- **Double-check your results.** Some analysis methods may produce partial or unexpected results depending on what information was available when you ran the analysis. If something seems wrong, [our debugging guide](./docs/walkthroughs/debugging-your-integration.md) can help you diagnose and debug your integration.
-
-- **Scan for issues and generate a compliance report.** Once your analysis is ready, we'll automatically queue an issue scan and report the results in the web application. Once an issue scan is complete, you can also [generate a report](https://docs.fossa.com/docs/running-a-scan) from the web application.
-
-- **Set up FOSSA in your CI.** You can also use your issue scan results as inputs to CI scripts. For GitHub repositories, you can use FOSSA's [native GitHub integration](https://docs.fossa.com/docs/automatic-updates#pull-request--commit-statuses-github-only) to report a status check on your PRs. For other CI integrations, you can also [use `fossa test`](docs/references/subcommands/test.md) to get programmatic issue status in CI.
-
-## User Manual
-
-For most users, Spectrometer will work out-of-the-box without any configuration. Just get an API key, run `fossa analyze`, and view your results in the FOSSA web application.
-
-Users who need more advanced customizations or features should see the [User Manual](./docs/README.md). Some common topics of interest include:
-
-- [Debugging your integration](./docs/walkthroughs/debugging-your-integration.md)
-- [Specifying vendored dependencies](docs/features/vendored-dependencies.md)
-- [Adding manual dependencies](docs/features/manual-dependencies.md)
-
-## Reporting Issues
-
-If you've found a bug or need support, the best way to get support is to email [support@fossa.com](mailto:support@fossa.com).
-
-Make sure to include reproduction steps and any relevant project files (e.g. `pom.xml`s, `package.json`s, etc.). Including the output from `fossa analyze --debug` in the email as well as any relevant fossa files (`fossa-deps.json`, `.fossa.yml`) will help expedite a solution.
-
-We'll try to respond to issues opened in this repository on a best-effort basis, but we mostly provide support via our [`support@`](mailto:support@fossa.com) email.
-
-## Contributing
-
-If you're interested in contributing, check out our [contributor documentation](./docs/contributing/README.md). PRs are welcome!
+Please migrate to using latest installation scripts provided at [fossa-cli](https://github.com/fossas/fossa-cli/).
```

Comment:
Can you make this link go directly to the portion of the fossa-cli repo about installation?